docs/topics/gradle.md

```diff
@@ -316,10 +299,43 @@ kotlin{
 
 </tabs>
 
-> You can use shorthand for a dependency on a Kotlin module, for example, kotlin("test") for "org.jetbrains.kotlin:kotlin-test".
+> You can use shorthand for a dependency on a Kotlin module, for example, kotlin('test') for "org.jetbrains.kotlin:kotlin-test".
 >
 {type="note"}
 
+You can choose JUnit 5 or TestNG by calling 
+[`useJUnitPlatform()`]( https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/Test.html#useJUnitPlatform) 
+or [`useTestNG()`](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/Test.html#useTestNG) in the test task:
+
+ <tabs>
+
+```groovy
+tasks {
+    test {
+        // enable TestNG support
+        useTestNG()
+        // or
+        // enable JUnit Platform (a.k.a. JUnit 5) support
+        useJUnitPlatform()
+    }
+}
+```
+
+```kotlin
```

Comment:
If Groovy and Kotlin snippets are the same, you can leave only one. We usually leave Groovy/